### PR TITLE
doc: VXLAN in IPv6 supports only for kernel > 3.12

### PR DESCRIPTION
**Pull request by cyclinder**
## Description

doc: VXLAN in IPv6 supports only for kernel > 3.12

## Related issues/PRs

https://github.com/projectcalico/calico/issues/6877
https://github.com/projectcalico/calico/issues/6273

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

File: calico/networking/vxlan-ipip.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Encapsulation of workload traffic is typically required only when traffic crosse
 **Limitations** 
 
 - IP in IP supports only IPv4 addresses
+- VXLAN in IPv6 supports only for kernel >= 3.12
```
Comment (mgleung) [inline calico/networking/vxlan-ipip.md:47]: nit: `VXLAN in IPv6 is only supported for kernel versions >= 3.12`

Comment (cyclinder) [reply calico/networking/vxlan-ipip.md:47]: sorry😄 I'm not a english native speaker. Now updated.

**Review by coutinhop — APPROVED** [review]
Thanks @cyclinder!

**Pull request merged**